FAQs.md

```diff
@@ -0,0 +1,165 @@
+# APlayer FAQs
+
+
+This is the list of Frequently asked questions for APlayer. It is intended to serve as a self-service utility document. This is to aid new users or already existing users of APlayer. This document may be updated from time to time to suit any new changes to the software as new developments or changes will be implemented.
+
+
+1. **What is APlayer?**
+
+APlayer is a  basic music player application for playing media(music) files on your mobile device.
+
+---
+
+2. **What are the features of APlayer?**
+
+APlayer is designed with all the basic features of a music player. Aside from that, it has some other cool features like; 
+
+- Playlist.
```

Comment:
i think desktop lyric and sleep timer  should be written here.

Reply:
done